tests/unit/pages/Login.spec.js

```diff
@@ -96,21 +96,40 @@ describe("Login.vue", () => {
         const requestOtp = jest
           .spyOn(UserAPIService, "requestOtp")
           .mockImplementation(() => jest.fn());
-
+        const startResendOTPTimer = jest.spyOn(
+          Login.methods,
+          "startResendOTPTimer"
+        );
         mountWrapper();
+
         await setPhoneNumber();
         await requestOTP();
+        await wrapper.setData({ isResendOTPEnabled: false });
 
         // resend OTP
         await wrapper.find('[data-test="resendOTP"]').trigger("click");
 
         await flushPromises();
 
         expect(requestOtp).toHaveBeenCalled();
+        expect(startResendOTPTimer).toHaveBeenCalled();
+        expect(wrapper.vm.resendOTPTimer).toBe(60);
         expect(wrapper.vm.resentOtp).toBe(true);
         expect(wrapper.vm.invalidOtp).toBe(false);
       });
 
+      it("OTP timer", async () => {
+        //To test the Resend OTP Timer
+        //Faketimer is only needed for this test case to advance the time of interval
+        jest.useFakeTimers();
+        wrapper.vm.startResendOTPTimer(1);
+        jest.advanceTimersByTime(1000);
```

Comment:
make the value of seconds that you passed above a variable (say, `numSeconds`) and reuse it here (`numSeconds * 1000`)

Reply:
Yes, this can be done.